components/link-list/src/_mixin.scss

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Gemeente Utrecht
+ * Copyright (c) 2021 Robbert Broersma
+ */
+
+@mixin reset-ul() {
+  margin-block-end: 0;
+  margin-block-start: 0;
+  margin-inline-end: 0;
+  margin-inline-start: 0;
+  padding-inline-start: 0;
+}
+
+@mixin reset-li() {
+  list-style: none;
+}
+
+@mixin utrecht-link-list {
+  --utrecht-link-text-decoration: none;
+  --utrecht-link-hover-text-decoration: underline;
+  --utrecht-link-focus-text-decoration: underline;
+
+  display: flex;
+  flex-direction: column;
+  margin-block-end: calc(var(--utrecht-space-around, 0) * var(--utrecht-link-list-margin-block-end, 0));
+  margin-block-start: calc(var(--utrecht-space-around, 0) * var(--utrecht-link-list-margin-block-start, 0));
+  row-gap: var(--utrecht-link-list-row-gap);
+}
+
+@mixin utrecht-link-list__item {
+  --utrecht-icon-size: var(--utrecht-link-list-icon-size);
+  --utrecht-link-icon-size: var(--utrecht-link-list-icon-size);
+  --utrecht-icon-inset-block-start: var(--utrecht-link-list-icon-inset-block-start);
+
+  display: block;
+  font-weight: var(--utrecht-link-list-item-font-weight);
+}
+
+@mixin utrecht-link-list__link {
+  align-items: baseline;
+  column-gap: var(--utrecht-link-list-item-column-gap);
```

Comment:
Should this be `--utrecht-link-list-link-column-gap` or should `column-gap` be set in the mixin for the `__item` instead?

Reply:
Also I'm wondering what the difference is between een link-list__item and a link-list__link... the React component also only implements the `LinkListLink`?

Reply:
you make some very good points

Reply:
done